src/AspNetCoreModuleV2/InProcessRequestHandler/inprocesshandler.cpp

```diff
@@ -59,6 +61,65 @@ IN_PROCESS_HANDLER::OnExecuteRequestHandler()
     return m_pApplication->OnExecuteRequest(m_pW3Context, this);
 }
 
+// static
+void * IN_PROCESS_HANDLER::operator new(size_t)
+{
+    DBG_ASSERT(sm_pAlloc != NULL);
+    if (sm_pAlloc == NULL)
+    {
+        return NULL;
```

Comment:
Not a fan of returning null, I'd like to stop to check allocations for NULL every time. Can you throw bad_alloc here please?

Reply:
yes, you can to try-catch on bad_alloc exception.

Reply:
Not, what I meant is can you throw bad_alloc if sm_pAlloc  is null, instead of returning null?

Reply:
for your comments about moving check createapplication failure to requesthandler. it is possible. It requires change for inprocess. Will do it with moving app_offline